nipoppy/workflows/dataset_status.py

```diff
@@ -0,0 +1,467 @@
+"""Workflow for status command."""
+
+import logging
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+from rich import box
+from rich.console import Console
+from rich.table import Table
+
+from nipoppy.env import LogColor, StrOrPathLike
+from nipoppy.tabular.bagel import STATUS_SUCCESS
+from nipoppy.workflows.base import BaseWorkflow
+
+# generate positive reinforcement emojis for data curation and processing
+# three-step process (least stringent to most stringent)
+# 1. start: (broom / rocket)
```

Comment:
How about changing "rocket" to "hourglass"? ⏳ I feel like that corresponds better with broom